packages/cozy-mespapiers-lib/src/components/ModelSteps/helpers.js

```diff
@@ -8,6 +8,9 @@
 export const isFileAlreadySelected = (formData, stepIndex, currentFile) => {
   for (const data of formData.data) {
     if (data.stepIndex === stepIndex) {
+      if (currentFile.constructor.name === 'Blob') {
+        return data.file.id === currentFile.id
```

Comment:
Same question here: is returning `false` early intended if the `id`s are different?

Reply:
Vu ensemble